oak_functions/abi/build.rs

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+extern crate prost_build;
+
+fn main() {
+    prost_build::compile_protos(&["../proto/abi.proto"], &["../proto"])
+        .expect("Proto compilation failed");
+}
```

Comment:
Now this is built twice as part of the loader though, right? I think we should remove the other build.rs file and just use the compiled protos from this ABI crate everywhere, WDYT?

Reply:
I removed the `build.rs` in the `loader`, but was it really being built twice? I thought the one in the `loader` only built `lookup_data.proto`.

Reply:
You are correct, it was not being built, but my point is that we should build all the protos in one build.rs script, either the ABI one or something else just for protos.

Reply:
Makes sense. So, now everything is being built in the `abi`.